### PR TITLE
add support for versions dropdown

### PR DESCRIPTION
**Pull request by sambible**
small airgun PR to add support for a new test: https://github.com/SatelliteQE/robottelo/pull/15268

### CODE REVIEW

**Review by vijaysawant — APPROVED** [review]
LGTM

File: airgun/entities/contentview_new.py [hunk 1]
```diff
@@ -129,6 +129,12 @@ def read_french_lang_cv(self):
         view.wait_displayed()
         return view.table.read()
 
+    def click_version_dropdown(self, entity_name, version, dropdown_option):
+        """Clicks a specific dropdown option for a CV Version"""
+        view = self.navigate_to(self, 'Version', entity_name=entity_name, version=version)
+        self.browser.plugin.ensure_page_safe(timeout='5s')
```
Comment (omkarkhatavkar) [inline airgun/entities/contentview_new.py:135]: Added some comnets in the robottelo PR.

**Comment by sambible** [comment]
@omkarkhatavkar updated this PR, and the robottelo PR - would appreciate a re-review :)

File: airgun/entities/contentview_new.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 from navmazing import NavigateToSibling
+import pytest
```
Comment (omkarkhatavkar) [inline airgun/entities/contentview_new.py:2]: no pytest in airgun

File: airgun/entities/contentview_new.py [hunk 3]
```diff
@@ -135,6 +137,20 @@ def click_version_dropdown(self, entity_name, version, dropdown_option):
         self.browser.plugin.ensure_page_safe(timeout='5s')
         return view.version_dropdown.item_select(dropdown_option)
 
+    def republish_metadata_error(self, entity_name, version):
+        """Clicks a specific dropdown option for a CV Version, that will throw an error"""
+        view = self.navigate_to(self, 'Version', entity_name=entity_name, version=version)
+        self.browser.plugin.ensure_page_safe(timeout='5s')
+        with pytest.raises(DropdownItemDisabled) as error:
```
Comment (omkarkhatavkar) [inline airgun/entities/contentview_new.py:146]:
```python
try:
    view.version_dropdown.item_select('Republish repository metadata')
except DropdownItemDisabled as error:
```

File: airgun/entities/contentview_new.py [hunk 4]
```diff
@@ -135,6 +137,20 @@ def click_version_dropdown(self, entity_name, version, dropdown_option):
...
+            view.version_dropdown.item_select('Republish repository metadata')
+        if (
+            'Item "Republish repository metadata" of dropdown ".//div[@data-ouia-component-id="cv-version-header-actions-dropdown"]" is disabled'
```
Comment (omkarkhatavkar) [inline airgun/entities/contentview_new.py:149]: please avoid mentioning the dropdown with xpath that does not help.

**Comment by LadislavVasina1** [comment]
Can we please run some PRT for this change @sambible?

**Review by chris1984 — APPROVED** [review]

**Review by LadislavVasina1 — APPROVED** [review]

**Pull request merged**